src/components/dashboard/FaceVerification/api/FaceVerificationApi.js

```diff
@@ -7,7 +7,7 @@ import Config from '../../../../config/config'
 import logger from '../../../../lib/logger/js-logger'
 import { unexpectedErrorMessage } from '../sdk/FaceTecSDK.constants'
 
-import { hideRedBoxIfNonCritical } from '../utils/kindOfTheIssue'
+import { hideRedBoxIfNonCritical } from '../utils/redBox'
```

Comment:
had to move `hideRedBoxIfNonCritical` to the separate file to break the circular dependency issue found by the upgraded eslint